files/scripts/swss.sh

```diff
@@ -292,6 +292,7 @@ start() {
     # Don't flush DB during warm boot
     if [[ x"$WARM_BOOT" != x"true" ]]; then
         debug "Flushing APP, ASIC, COUNTER, CONFIG, and partial STATE databases ..."
+        clean_up_tables APPL_DB "'PORT_TABLE:Ethernet*'"
```

Comment:
I think xcvrd should subscribe to STATE_DB instead of APP_DB. This should not be done here, IMO

Reply:
@mihirpat1 i see Xcvrd subscribing to these tables. What am I missing?

```
  port_tbl_map = [
        {'CONFIG_DB': swsscommon.CFG_PORT_TABLE_NAME},
        {'STATE_DB': 'TRANSCEIVER_INFO'},
        {'STATE_DB': 'PORT_TABLE', 'FILTER': ['host_tx_ready']},
    ]
```

Reply:
@prsunny - XCVRD already subscribes to APPL_DB since with this change, I am changing the way to delete entries from APPL_DB. Am I missing something here?
https://github.com/sonic-net/sonic-platform-daemons/blob/502c0b6622008363cb1ed6d1b7c85b4093997093/sonic-xcvrd/xcvrd/xcvrd.py#L1173

Also, as @prgeor mentioned, XCVRD already subscribes to STATE_DB as well.